.github/workflows/dangerjs-checks.yml

```diff
@@ -0,0 +1,26 @@
+name: DangerJS PR Checks
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  danger-checks:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the Repo
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
`fetch-depth` can be removed. This is only needed for workflows that require full git history of branches / tags etc (like changesets)

Reply:
Done!